### DISCUSSION TITLE
Help using and maybe modifying an extension

### DISCUSSION THREAD

**Discussion by barryburd** [microsoft/vscode-discussions#1294] [Extension Development QnA] [answered]
I'm hoping to use [Litoshow](https://marketplace.visualstudio.com/items?itemName=FernandoPalacios.litoshow) as part of a computer science course that I'm teaching in September. I have questions about the extension. But, so far, I haven't been able to get the developer's attention. (That's understandable, by the way. I'm not complaining.) I've never written extensions for VSCode, and I'd rather not take hours to learn VSCode Extension programming for this project alone. (I'm busy too.) Can anyone offer to answer a few questions after looking at the extension's [source code](https://github.com/fernandops26/litoshow-vscode-extension)? My first question is, can students use Litoshow to record their work, and send me an artifact so I can replay their work?

**Comment by alefragnani** [comment, marked as answer]
Hi @barryburd ,

I see no reference to _export / import_ of recordings, so I assume students won't be able to send you any artifact. But I would suggest you explore the extension a bit deeper, looking for commands and settings other than those documented in the `readme` file. Maybe the author may have forgot to update it. 

Just a side note, since the extension has few downloads and is not updated in the last two years, if you don't receive answers from the author, I would suggest you to look for other (more active) extensions.

But, if you really need use this extension, and depending on how much more answers you have about it, or even updates on the source code, maybe you will need to look for help on additional locations as well, such as [Stack Overflow](https://stackoverflow.com/questions/tagged/vscode) or [Slack](https://aka.ms/vscode-dev-community). This discussions repo is focused for extension development for now.

Hope this helps

**Reply by barryburd** [reply]
Thank you, alefragnani. This extension is the best one I've found for the purpose I have in mind. My guess is that there's a way to tweak this extension so that it saves its work in an exportable file. But I don't know enough about programming VSCode extensions to figure out where to start in the source code in GitHub. Since this repo is focused on extension development, I figured it would be a good place to ask for help.

**Reply by alefragnani** [reply]
I also think it may possible, based on what I could see in its source code. 

A VS Code extension could be as simple as cloning and running `npm i` to install dependencies, and clicking Debug inside VS Code, as detailed here (https://code.visualstudio.com/api/get-started/your-first-extension). And except for the Vite pieces used for a WebView, this extension appears to be simple. I don't know Vite, but if the scripts defined in `package,json` works, it should be as easy as this.

I suggested looking for "additional places" for such help, if you don't have success here. Because this discussion repo is more focused on extension authors looking for help/info on their extensions, not exactly using an extension, which we also tries to do 😁 

Btw, are you willing to fork and publish that extension yourself once these updates are made, or would you open a PR to the original author?
